baggage-processor/src/test/java/io/opentelemetry/contrib/baggage/processor/BaggageProcessorCustomizerTest.java

```diff
@@ -38,57 +45,69 @@
 import org.mockito.junit.jupiter.MockitoExtension;
 
 @ExtendWith(MockitoExtension.class)
-class BaggageSpanProcessorCustomizerTest {
+class BaggageProcessorCustomizerTest {
 
   private static final String MEMORY_EXPORTER = "memory";
 
   @Test
   void test_customizer() {
     assertCustomizer(Collections.emptyMap(), span -> span.hasTotalAttributeCount(0));
-    assertCustomizer(
-        Collections.singletonMap(
-            "otel.java.experimental.span-attributes.copy-from-baggage.include", "key"),
-        span -> span.hasAttribute(AttributeKey.stringKey("key"), "value"));
+    Map<String, String> properties = new HashMap<>();
+    properties.put("otel.java.experimental.span-attributes.copy-from-baggage.include", "key");
+    properties.put("otel.java.experimental.log-attributes.copy-from-baggage.include", "key");
+    // TODO try use
+    //  AttributeAssertion attributeAssertion =
```

Comment:
@zeitlinger do you know if we could use an ÀttributeAssertion`on both span and log record data? I couldn't find a solution.

Reply:
```java
 assertThat(finishedLogRecordItems)
                  .hasSize(1)
                  .anySatisfy(
                      logRecord -> OpenTelemetryAssertions.assertThat(logRecord.getAttributes())
                          .containsEntry(AttributeKey.stringKey("key"), "value"));
```